app/consumers/compliance_reports_consumer.rb

```diff
@@ -42,14 +42,14 @@ def download_file
   end
 
   def enqueue_job
-    if validate == 'success'
-      logger.info "Received message, enqueueing: #{@msg_value}"
+    return unless validate == 'success'
+
+    logger.info "Received message, enqueueing: #{@msg_value}"
+    @report_contents.each do |report|
       job = ParseReportJob.perform_async(
-        ActiveSupport::Gzip.compress(@report_contents), @msg_value
+        ActiveSupport::Gzip.compress(report), @msg_value
       )
       logger.info "Message enqueued: #{message_id} as #{job}"
-    else
-      logger.error "Error parsing report: #{message_id}"
```

Comment:
Notice this is the *exact* same error as the one that's logged in `validation_message`  if there's a failure. It makes no sense to log the same thing twice.